pyvirtualcam/native_macos/main.mm

```diff
@@ -0,0 +1,165 @@
+// Rest of files is stolen from:
```

Comment:
I should probably add my own copyright at the top of this file

Reply:
Sure, go for it.

Reply:
Done. Added MIT license - hopefully it's clear enough that my copyright is only for main.mm - the other files contain different license headers; so people hopefully understand.